ccl/workloadccl/allccl: TestAllRegisteredSetup failed

ccl/workloadccl/allccl.TestAllRegisteredSetup [failed](https://mesolite.cluster.engflow.com/invocations/default/993761d9-28d8-4e80-be72-ba3284aa29d6?testReportRun=2&testReportShard=3&testReportAttempt=1#targets-Ly9wa2cvY2NsL3dvcmtsb2FkY2NsL2FsbGNjbDphbGxjY2xfdGVzdA==) on release-25.4.11-rc @ [5175d7571f3d6bf9605fe3595da7579f26ab3342](https://github.com/cockroachdb/cockroach/commits/5175d7571f3d6bf9605fe3595da7579f26ab3342):


```
=== RUN   TestAllRegisteredSetup
--- FAIL: TestAllRegisteredSetup (105.08s)
=== RUN   TestAllRegisteredSetup/ycsb
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestAllRegisteredSetup_ycsb69706706
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:154: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    all_test.go:227: failed insert into usertable: sql: database is closed
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad.func2.1
          | 	pkg/workload/workloadsql/dataload.go:139
          | github.com/cockroachdb/cockroach/pkg/workload/workloadsql.InsertsDataLoader.InitialDataLoad.func2
          | 	pkg/workload/workloadsql/dataload.go:167
          | golang.org/x/sync/errgroup.(*Group).Go.func1
          | 	external/org_golang_x_sync/errgroup/errgroup.go:93
          | runtime.goexit
          | 	src/runtime/asm_amd64.s:1700
        Wraps: (2) failed insert into usertable
        Wraps: (3) sql: database is closed
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errors.errorString
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestAllRegisteredSetup_ycsb69706706
--- FAIL: TestAllRegisteredSetup/ycsb (63.77s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #169034 ccl/workloadccl/allccl: TestAllRegisteredSetup failed [C-test-failure O-robot T-testeng branch-release-25.4 branch-release-25.4.10-rc target-release-25.4.11]
- #168946 ccl/workloadccl/allccl: TestAllRegisteredSetup failed [C-test-failure O-robot T-testeng branch-release-25.4 target-release-25.4.11]
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAllRegisteredSetup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-64236

--- a/pkg/workload/all/all_test.go
+++ b/pkg/workload/all/all_test.go
@@ -222,6 +222,12 @@ func TestAllRegisteredSetup(t *testing.T) {
 			sqlutils.MakeSQLRunner(srv.SystemLayer().SQLConn(t)).Exec(t, `SET CLUSTER SETTING kv.range_merge.queue.enabled = false`)
 
 			var l workloadsql.InsertsDataLoader
+			if meta.Name == `ycsb` {
+				// YCSB's default 1000-row insert batches fan out to ~11K CPuts
+				// because each column lives in its own family. Under external
+				// process tenants this can starve SQL liveness heartbeats.
+				l.BatchSize = 100
+			}
 			if _, err := workloadsql.Setup(ctx, db, gen, l); err != nil {
 				t.Fatalf(`%+v`, err)
 			}

